app/views/toolbar.js

```diff
@@ -41,6 +41,16 @@ const Toolbar = Mn.LayoutView.extend({
         this.listenTo(application.channel, 'search:done', () => {
             this.ui.search.removeClass('loading');
         });
+
+        this.listenTo(application.channel, 'drawer:toggle', () => {
```

Comment:
Here I still don't understand why puting this toggles here instead of the view's parent, 'cause every handlers are just bound to events…